group_vars/all.yml

```
@@ -1682,6 +1682,9 @@ varnish_purge_logfile: "{{ varnish_log_dir }}/varnish-purge.log"
 # Error page customization
 varnish_error_page_color_primary: "#444"
 
+#Send original URL to backend
+varnish_original_url: true
```

Let's keep it disabled by default, we only need this in special cases
```suggestion
varnish_original_url: false
```